### PR TITLE
B3

### PR DESCRIPTION
**Pull request by TheSnakeWitcher**

### CODE REVIEW

File: file5.ts [hunk 1]
```diff
@@ -1,7 +1,3 @@
 function show01() {
     console.log("some")
-  }
-  
-  function main3() {
-      show01()
-  }
```
Comment (SmilingWitch) [inline file5.ts:7]: Por que lo borraste

File: file2.ts [hunk 2]
```diff
@@ -1,7 +1,3 @@
 function show2() {
- console.log("some")
-}
-  
-function main() {
-    show2()
+    console.log("some")
```
Comment (SmilingWitch) [inline file2.ts:2]: quiero que diga 
```ts
console.log("Log")
```

**Review by SmilingWitch — CHANGES_REQUESTED** [review]
do it do it do it

File: file5.ts (hunk 1 above)
Comment (TheSnakeWitcher) [reply file5.ts]: Fixed in [344e5ea](https://github.com/TheSnakeWitcher/nena-training/pull/6/commits/344e5ea2cef1b60af58ba5edad868563ac8d0885)

File: file2.ts (hunk 2 above)
Comment (TheSnakeWitcher) [reply file2.ts]: Fixed in [6da6a06](https://github.com/TheSnakeWitcher/nena-training/pull/6/commits/6da6a0628bb4b396978b7f46184f6ce50574458e)

**Review by SmilingWitch — APPROVED** [review]

**Pull request merged**